src/Microsoft.AspNet.Session/SessionServiceCollectionExtensions.cs

```diff
@@ -7,8 +7,16 @@
 
 namespace Microsoft.Framework.DependencyInjection
 {
+    /// <summary>
+    /// Extension methods for adding session services to the DI container.
+    /// </summary>
     public static class SessionServiceCollectionExtensions
     {
+        /// <summary>
+        /// Adds services the session state middleware.
```

Comment:
Sorry, i think it should be **Adds services required for application session state.**, let me know if sounds is good so I will update the PR now